pfb-analysis/connectivity/census_blocks.sql

```diff
@@ -96,7 +96,7 @@ ALTER TABLE neighborhood_census_blocks DROP COLUMN IF EXISTS tmp_geom_buffer;
 ALTER TABLE neighborhood_census_blocks ADD COLUMN tmp_geom_buffer geometry(multipolygon, :nb_output_srid);
 
 UPDATE  neighborhood_census_blocks
-SET     tmp_geom_buffer = ST_Multi(ST_Buffer(geom,50));
+SET     tmp_geom_buffer = ST_Multi(ST_Buffer(geom,15));
```

Comment:
Since we technically allow any SRID to be run, not just ones in meters, these magic values should probably parameterized and passed to this script in the same way as the values recently modified in other scripts, using a helpful variable name added to run_connectivity.sh.